app/models/connector/IFSApiName.scala

```diff
@@ -25,14 +25,16 @@ sealed abstract class IFSApiName(override val entryName: String) extends EnumEnt
 object IFSApiName extends Enum[IFSApiName] {
   val values = IndexedSeq[IFSApiName]()
 
-  case object Api1171 extends IFSApiName("1171")
-  case object Api1638 extends IFSApiName("1638")
-  case object Api1639 extends IFSApiName("1639")
-  case object Api1640 extends IFSApiName("1640")
-  case object Api1786 extends IFSApiName("1786")
-  case object Api1802 extends IFSApiName("1802")
-  case object Api1803 extends IFSApiName("1803")
-  case object Api1894 extends IFSApiName("1894")
-  case object Api1895 extends IFSApiName("1895")
-  case object Api1965 extends IFSApiName("1965")
+  case object CitizenDetails extends IFSApiName("citizen-details")
```

Comment:
hmm, it is not an IFS endpoint it is a bit strange to keep it as part of IFSApiName. Does it need to be part of this enum for convinience? Maybe we just need to rename the enum